plus/tool/src/main/resources/Messages.properties

```diff
@@ -93,3 +93,4 @@ plus.tool.add=Add Tenant
 plus.tool.on=On
 plus.tool.off=Off
 
+plus.tenant.form.issuer.error.message=Issuer should not end with a trailing slash /
```

Comment:
I would use a different key for this property. It is very general. Maybe you could use "plus.issuer.error.slash" ? We know that all the strings in this file are messages already, so I don't think you need to use ".message" in your keyname.

Reply:
LGTM, I will change it to ```plus.tenant.issuer.error.slash```.